src/elements/content-sidebar/AddTaskMenu.scss

```diff
@@ -19,9 +19,11 @@ $bcs-AddTaskMenu-width: 256px;
 }
 
 .bcs-AddTaskMenu-title {
-    color: $bdl-neutral-01;
+    font-weight: 700;
 }
 
 .bcs-AddTaskMenu-description {
-    color: $bdl-neutral-02;
+    color: $nines;
```

Comment:
we should map this to `$bdl-gray-62` for a11y compliance

Reply:
Done